guidelines/terms/22/cognitive-function-test.html

```diff
@@ -5,12 +5,11 @@
    					
    <p>A task that requires the user to remember, manipulate, or transcribe information. Examples include, but are not limited to:</p>
    <ul>
-       <li>memorization, such as remembering a username, password, set of characters, images, or patterns;</li>
+       <li>memorization, such as remembering a username, password, set of characters, images, or patterns. Common personal identifiers such as name, e-mail, phone number or handle on social media are not considered memorization as they are consistent across websites;</li>
```

Comment:
Should be "phone number, and handle on social media", making this an "and" list since all could apply and also adding the Oxford comma since we do everywhere else for clarity.

Reply:
Thanks, also not sure why the understanding doc didn't match, I'm sure I did both at the same time.